arkane/statmech.py

```diff
@@ -53,12 +53,12 @@
 
 from arkane.common import ArkaneSpecies, symbol_by_number, get_principal_moments_of_inertia
 from arkane.encorr.corr import get_atom_correction, get_bac
-from arkane.logs.gaussian import GaussianLog
-from arkane.logs.log import Log
-from arkane.logs.molpro import MolproLog
-from arkane.logs.orca import OrcaLog
-from arkane.logs.qchem import QChemLog
-from arkane.logs.terachem import TeraChemLog
+from arkane.ess.gaussian import GaussianLog
+from arkane.ess.log import Log
+from arkane.ess.molpro import MolproLog
+from arkane.ess.orca import OrcaLog
+from arkane.ess.qchem import QChemLog
+from arkane.ess.terachem import TeraChemLog
```

Comment:
Since you added imports to the `__init__.py` file, these could potentially be replaced with
```
from arkane.ess import GaussianLog, Log, MolproLog, OrcaLog, QChemLog, TeraChemLog
```

It's also fine as is. Completely up to you.

Reply:
I agree. It's easier to maintain. Modified and force pushed.